src/model/index.ts

```diff
@@ -21,6 +21,14 @@ import CategoricalNumberColumn from './CategoricalNumberColumn';
 import NestedColumn from './NestedColumn';
 import DummyColumn from './DummyColumn';
 import LinkColumn from './LinkColumn';
+import HeatmapColumn from './HeatmapColumn';
```

Comment:
as far as I understood, all of the multi value columns are the same but just use a different default renderer. So, it could be a single model class where the user can specify a different default renderer